games/widelands/Portfile

```diff
@@ -1,19 +1,17 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
 
 PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           cxx11 1.0
-
 name                widelands
-version             build18
+conflicts           widelands-devel
```

Comment:
Hah - I know. There was supposed to be a widelands-devel <https://trac.macports.org/ticket/52542>, but it got bogged down waiting to be committed, and then the final version of 19 came through instead so I asked to have it closed out yesterday. Some widelands users are still asking for a devel version -  for build20 now, though...